go/libraries/doltcore/doltdb/doltdb.go

```diff
@@ -1278,10 +1278,7 @@ func (ddb *DoltDB) PushChunksForRefHash(ctx context.Context, tempDir string, src
 func (ddb *DoltDB) PullChunks(ctx context.Context, tempDir string, srcDB *DoltDB, stRef types.Ref, progChan chan datas.PullProgress, pullerEventCh chan datas.PullerEvent) error {
 	if datas.CanUsePuller(srcDB.db) && datas.CanUsePuller(ddb.db) {
 		puller, err := datas.NewPuller(ctx, tempDir, defaultChunksPerTF, srcDB.db, ddb.db, stRef.TargetHash(), pullerEventCh)
-
-		if err == datas.ErrDBUpToDate {
-			return nil
-		} else if err != nil {
+		if err != nil {
```

Comment:
I'm worried this introduces problems. I haven't checked, but you need to examine every place that calls this method and make sure they have equivalent error checking code to what you removed, somewhere in the call stack. Otherwise this error will interfere with other operations.

Reply:
Looks like FetchTag has this problem.